### ISSUE TITLE
Update Actions List Endpoint documentation

### ISSUE THREAD

**Issue by ricardocosta** [looker/actions#244]
The current documentation for the [Actions List Endpoint](https://github.com/looker/actions/blob/master/docs/action_api.md#actions-list-endpoint) seems to be outdated:

- `supported_download_settings` seems to be an `array` now, instead of `boolean`.
- `csv_zip` is missing in the `supported_formats` values list.

I'm looking at the [types definitions](https://github.com/looker/actions/blob/master/src/api_types/integration.ts) to know the correct action configuration.

**Comment by abatrinu** [comment]
Yup ran into same issues

**Comment by joerzepiejewski** [comment]
yup, run into the same issue... An example I've received from the Looker folks was something like:
```
      "supported_download_settings": [
        "push"
      ]
```

I'm looking for all the enum values for it
